src/main/kotlin/uk/gov/dluhc/printapi/messaging/service/PrintResponseProcessingService.kt

```diff
@@ -107,5 +114,38 @@ class PrintResponseProcessingService(
         }
 
         certificateRepository.save(certificate)
+
+        sendEmailIfNotDelivered(newStatus, certificate, printResponse.requestId)
     }
+
+    private fun sendEmailIfNotDelivered(newStatus: Status, certificate: Certificate, printResponseRequestId: String) {
```

Comment:
- It will be clear to rename this to `sendEmailIfCertificateNotDelivered`. 
By just looking at this fun name, I initially thought that we are resending the email if email wasn't delivered

- Also please change exception log message to be `failed to send Certificate Not Delivered email .....`.
- Is there any fall back plan as that error message might get lost in the system (not a blocker) when emails are not sent?